### PR TITLE
Display warning on mobile browsers

### LINKED ISSUE #397: Add copy  that we're not supporting mobile
### Description

Detect Mobile Browsers and Display Warning Dialogue "We Strongly Recommend Playing This on Something with a Keyboard"

### Screenshots

_No response_

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by polypixeldev**
This works by parsing the user agent with a regex from detectmobilebrowsers.com. I'm not sure if this is the best way, but it works.

Closes #397

### CODE REVIEW

**Comment by polytroper** [comment]
Indeed, will merge if it seems like mobile ain't gonna happen before TC.

File: App/main.js [hunk 1]
```diff
@@ -96,6 +97,25 @@ const editor = Editor(ui)
 ui.levelText.setAttribute('hide', true)
 ui.veil.setAttribute('hide', true)
 
+// Detect mobile devices and show the warning
+const ua = navigator.userAgent || navigator.vendor || window.opera
+let mobile = false
+
+if (
+  /(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(
```
Comment (albertchae) [inline App/main.js:105]: Nice, I might use this instead of https://github.com/hackclub/sinerider/pull/320/commits/cfaa3cd94ca91179711d7c5637dbd16eff2a28b6

**Comment by polypixeldev** [comment]
Closing as it seems this is no longer needed

**Comment by alhardwarehyde** [comment]
@polytroper think we need to get this merged for launch bc pulling mobile support

**Comment by camdan-me** [comment]
Why not just detect screen dimensions? It seems like the issue is more a matter of screen size than actually using a mobile device. Or maybe catch both, if the issue is more surrounding touchscreens?

**Comment by karamassie** [comment]
sounds like this is done @camdan-me . we can assess a better solution post launch

**Comment by polytroper** [comment]
@camdan-me nein, you can use a portrait device mostly fine with some occasional issues as long as you have a keyboard. It is specifically the mobile keyboard that is the issue.

**Pull request merged**